### DISCUSSION TITLE
Cerco una soluzione efficace ed economica

### DISCUSSION THREAD

**Discussion by Ziobirrone** [indomus/forum#3261] [Hardware & Firmware]
Salve a tutti, sono Fabrizio, un appasionato Homebrewer e vorrei rendere domotica la mia cella di fermentazione.
Sto cercando da tempo un termostato che gestica sia caldo che freddo e che possa essere controllato tramite wifi. 
Fino ad oggi o solo trovato swich con sonda di tempreratura, che ovviamente non mi danno la lettura in loco ma solo tramite smartphone. I temostati che ho trovato online sono tutti con 1 solo relè e quindi non riesco a gestire caldo freddo in modo separato.
Sarei anche disposto ad avere un termostato da gestire in loco e un switch con 2 relè da gestire con delle schene.
L'unica cosa che non so e se tramite le scene posso gestire solo la temperatura rilevata op. posso anche gestire la temperatura impostata.
Per spiegarmi provo a fare un esempio:
Imposto la cella di fermentazione a 20° tramite un termostato.
Se la temperatura rilevata (es. 22°) è > della temperatura impostata (20°) mi deve accendere il relè 1 (compressore frigorifero)
Se la temperatura rilevata (es. 18°) è < della temperatura impostata (20°) mi deve accendere il relè 2 (Serpentina riscaldante)

Sono disponibile ad ofrivirvi un paio delle mie birre se mi date una possibile soluzione (anche con prodotti "cinesi")

Grazie in anticipo

ps: se serve posso anche aggiungere che so programmare con qualche linguaggio pc e ho già creato, anche,  degli sketchs per arduino e ESP32.

**Comment by indomus** [comment]
> Salve a tutti, sono Fabrizio, un appasionato Homebrewer e vorrei rendere domotica la mia cella di fermentazione. Sto cercando da tempo un termostato che gestica sia caldo che freddo e che possa essere controllato tramite wifi. Fino ad oggi o solo trovato swich con sonda di tempreratura, che ovviamente non mi danno la lettura in loco ma solo tramite smartphone. I temostati che ho trovato online sono tutti con 1 solo relè e quindi non riesco a gestire caldo freddo in modo separato. Sarei anche disposto ad avere un termostato da gestire in loco e un switch con 2 relè da gestire con delle schene. L'unica cosa che non so e se tramite le scene posso gestire solo la temperatura rilevata op. posso anche gestire la temperatura impostata. Per spiegarmi provo a fare un esempio: Imposto la cella di fermentazione a 20° tramite un termostato. Se la temperatura rilevata (es. 22°) è > della temperatura impostata (20°) mi deve accendere il relè 1 (compressore frigorifero) Se la temperatura rilevata (es. 18°) è < della temperatura impostata (20°) mi deve accendere il relè 2 (Serpentina riscaldante)

metti su [Home Assistant](https://indomus.it/home-assistant/), affiacagli un qualsiasi sensore ambientale zigbee e uno switch shelly o sonoff e fai la qualunque. quello che hai descritto e ben altro.

**Reply by Ziobirrone** [reply]
non conosco Home Assistant dove lo dovrei mettere?

**Reply by KC-inDomus** [reply]
ti abbiamo messo un link apposta...

**Reply by Ziobirrone** [reply]
Grazie dell'imbeccata ma , qualcosa di più semplice? Magari un termostato caldo/freddo wifi con schermo per impostare la temperatura in loco...? Non ho bisogno di gestire tutta la casa ma solo 2 relè
